src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.Session_FilterModel.cs

```diff
@@ -210,7 +210,9 @@ internal partial class Session
                 // matches the text typed so far.
 
                 // Ask the language to determine which of the *matched* items it wants to select.
+                // We have a Document so GetCompletionService should always succeed.
                 var service = this.Controller.GetCompletionService();
+                Contract.ThrowIfNull(service, nameof(service));
```

Comment:
This is completely unsafe.  We're calling on the background, and the stuff may be unmapped on the FG.

We should either capture hte service initially and pass it through, or we have to be resilient to it not being aailable here.